setup.py

```diff
@@ -41,38 +28,27 @@ def extract_version():
 LICENSE = read('LICENSE')
 long_description = read('README')
 
-attributes = dict(name='gsw',
-                  version=extract_version(),
-                  packages=['gsw', 'gsw/tests', 'gsw/ice'],
-                  author=['Eric Firing', 'Filipe Fernandes'],
-                  author_email='efiring@hawaii.edu',
-                  description='Gibbs Seawater Oceanographic Package of TEOS-10',
-                  long_description=long_description,
-                  license=LICENSE,
-              #url='https://github.com/TEOS-10/python-gsw',
-              #download_url='https://pypi.python.org/pypi/gsw/',
-              platforms='any',
-              keywords=['oceanography', 'seawater', 'TEOS-10',],
-              #install_requires=['numpy'],
-              #tests_require=tests_require,
-                  )
-
-def configuration(parent_package='', top_path=None):
-
-    config = Configuration('gsw',
-                           parent_package,
-                           top_path,
-                           **attributes)
-
-    config.add_extension('_gsw_ufuncs',
-                         ['src/_ufuncs.c',
-                          'src/c_gsw/gsw_oceanographic_toolbox.c',
-                          'src/c_gsw/gsw_saar.c',
-                          ],
-                         include_dirs=['src/c_gsw'],
-                         )
-
-    return config
-
-if __name__ == "__main__":
-    setup(configuration=configuration)
+config = dict(
+    name='gsw',
+    version=extract_version(),
+    packages=['gsw', 'gsw/tests', 'gsw/ice'],
+    author=['Eric Firing', 'Filipe Fernandes'],
+    author_email='efiring@hawaii.edu',
+    description='Gibbs Seawater Oceanographic Package of TEOS-10',
+    long_description=long_description,
+    license=LICENSE,
+    # url='https://github.com/TEOS-10/GSW-python',
+    # download_url='https://pypi.python.org/pypi/gsw/',
+    platforms='any',
+    keywords=['oceanography', 'seawater', 'TEOS-10'],
+    setup_requires=['numpy'],
+    ext_modules=[
+        Extension('gsw._gsw_ufuncs',
```

Comment:
The 3.6 builds can't be imported because `_gsw_ufuncs` is not being found.  I assume that is either because The Extension here has the `gsw.` prefix, or because numpy.distutils does something differently--those seem to be the relevant differences between my original and your modification.